main.go

```diff
@@ -19,8 +19,9 @@ func main() {
 	http.HandleFunc("/pokemon", nc.GetPokemon)
 	http.HandleFunc("/index", nc.PokemonIndex)
 	http.HandleFunc("/save", nc.SavePokeApi)
+	http.HandleFunc("/concurrentRead", nc.ConcurrentRead)
 
-	err := http.ListenAndServe(":8080", nil) // implement graceful shutdown
```

Comment:
This may not be a graceful shutdown. the, if err below, would only happen when server failed to initialize

Reply:
Ups sorry!! I meant to implement graceful shutdown and left the comment to remember, but I forgot anyway 😅